azext_edge/edge/params.py

```diff
@@ -135,7 +135,14 @@ def load_iotops_arguments(self, _):
             "Choose 0 for a summary view (minimal output), "
             "1 for a detailed view (more comprehensive information), "
             "or 2 for a verbose view (all available information).",
-        )
+        ),
+        context.argument(
+            "resource_name",
+            options_list=["--resource-name", "--rn"],
```

Comment:
this param should have filter in its name like: `--resource-name-filter` and `--rnf`

Otherwise, a customer could use this param (not read the param help) and then assume that other params should support filtering too

Reply:
yeah good point, will update!

Reply:
actually after carefully thinking, I don't think filter is necessary since all other params in this command is actually working as filters as well, it's a bit strange to only add filter in the param?